psc-ide-server/Main.hs

```diff
@@ -43,19 +36,19 @@ import           Network                           hiding (socketPort, accept)
 import           Network.BSD                       (getProtocolNumber)
 import           Network.Socket                    hiding (PortNumber, Type,
                                                     sClose)
-import           Options.Applicative
+import           Options.Applicative               hiding ((<>))
 import           System.Directory
 import           System.FilePath
-import           System.IO
+import           System.IO                         hiding (putStrLn, print)
 import           System.IO.Error                   (isEOFError)
 
 import qualified Paths_purescript                  as Paths
 
 -- "Borrowed" from the Idris Compiler
```

Comment:
Just a note: we should definitely try to give proper attribution in the license file if this is indeed taken from the Idris compiler. It might be simpler to just use something like http://hackage.haskell.org/package/network-simple-0.4.0.5/docs/Network-Simple-TCP.html though.